glide.yaml

```diff
@@ -6,5 +6,8 @@ import:
   - context
 - package: google.golang.org/grpc
 - package: k8s.io/kubernetes
+  vcs: git
+  repo: git@github.com:kubernetes/kubernetes.git
```

Comment:
Yes, it works perfectly and it's the only way to get these sources from git, not from your local GOPATH.